### ISSUE TITLE
"Size must be a power of 2" on iPhone 11 Pro

### ISSUE THREAD

**Issue by Bcow7** [jscalo/tempi-fft#14]
There's an assert failure "Size must be a power of 2" when instantiating the FFT on iPhone 11 Pro. I can replicate it with iPhone 11 Pro simulator. 

Any idea of what is going wrong on this new device? Is it because of new CPU?

**Comment by jcutting7** [comment]
Yep - I have the same issue. It is because the number of samples returned by the microphone is only around ~940 or 941. 

One thing I just tried was to pass 1024 as a fixed size to the initializer, rather than the size of my sample, i.e.
`TempiFFT(withSize: 1024, sampleRate: Constants.AUDIO_SAMPLE_RATE)`
instead of
`TempiFFT(withSize: CMSampleBufferGetNumSamples(sampleBuffer), sampleRate: Constants.AUDIO_SAMPLE_RATE)`

Although it runs without getting errors, unfortunately the bins give strange results. The hanning or something isn't right. My visualizer I had made just isn't right any more.
